gnat2goto/driver/goto_utils.adb

```diff
@@ -139,6 +153,32 @@ package body GOTO_Utils is
          (I_Subtype => Base,
           Width => Pointer_Type_Width));
 
+   function Make_Array_String_Type (Size : Integer) return Irep is
```

Comment:
Maybe `Make_Constant_Length_String_Type` or something, because this is specifically for known-length strings